tt_metal/tools/profiler/kernel_profiler.hpp

```diff
@@ -25,6 +25,8 @@
 #define PROFILER_MSG __FILE__ "," $Line ",KERNEL_PROFILER"
 #define PROFILER_MSG_NAME( name )  name "," PROFILER_MSG
 
+#define PROFILE_DISPATCH_KERNEL (defined(DISPATCH_KERNEL) && defined(COMPILE_FOR_NCRISC) && (PROFILE_KERNEL == PROFILER_OPT_DO_DISPATCH_CORES))
```

Comment:
Where you planning to use this in the template?

Reply:
oh whoops, originally moved this out because I was going to do the same check in multiple places but that's not needed anymore. Will remove